src/torrent/tracker.h

```diff
@@ -159,7 +159,7 @@ class LIBTORRENT_EXPORT Tracker {
 
 inline bool
 Tracker::can_request_state() const {
-  return !(is_busy() && latest_event() != EVENT_SCRAPE) && is_usable();
+  return (!is_busy() || latest_event() == EVENT_SCRAPE) && is_usable();
```

Comment:
The way these are written is intentional, it tells the reader what the meaning of the check is.

These should not be reduced.